lib/samples/apply_simple_renderer_to_feature_layer/apply_simple_renderer_to_feature_layer_sample.dart

```diff
@@ -27,69 +27,94 @@ class ApplySimpleRendererToFeatureLayerSample extends StatefulWidget {
 
 class _ApplySimpleRendererToFeatureLayerSampleState
     extends State<ApplySimpleRendererToFeatureLayerSample> {
-  final _mapViewController = ArcGISMapView.createController();
   late FeatureLayer _featureLayer;
-  final _map = ArcGISMap.withBasemapStyle(BasemapStyle.arcGISTopographic);
-
-  @override
-  void initState() {
-    super.initState();
-
-    final uri = Uri.parse(
-        'https://services.arcgis.com/V6ZHFr6zdgNZuVG0/arcgis/rest/services/Landscape_Trees/FeatureServer/0');
-    final serviceFeatureTable = ServiceFeatureTable.withUri(uri);
-    _featureLayer = FeatureLayer.withFeatureTable(serviceFeatureTable);
-    _map.operationalLayers.add(_featureLayer);
-    _map.initialViewpoint = Viewpoint.fromCenter(
-      ArcGISPoint(
-        x: -9177250,
-        y: 4247200,
-        spatialReference: SpatialReference.webMercator,
-      ),
-      scale: 9500,
-    );
-    _mapViewController.arcGISMap = _map;
-  }
+  final _mapViewController = ArcGISMapView.createController();
+  var _ready = false;
+  var _usingDefaultRenderer = true;
 
   @override
   Widget build(BuildContext context) {
     return Scaffold(
       body: SafeArea(
-        child: Column(
+        top: false,
+        child: Stack(
           children: [
-            Expanded(
-              child: ArcGISMapView(
-                controllerProvider: () => _mapViewController,
-              ),
-            ),
-            Row(
+            Column(
               children: [
-                const Spacer(),
-                TextButton(
-                  onPressed: resetRenderer,
-                  child: const Text('Reset'),
-                ),
-                const Spacer(),
-                TextButton(
-                  onPressed: overrideRenderer,
-                  child: const Text('Override'),
+                Expanded(
+                  child: ArcGISMapView(
+                    controllerProvider: () => _mapViewController,
+                    onMapViewReady: onMapViewReady,
+                  ),
                 ),
-                const Spacer(),
+                SizedBox(
```

Comment:
@kmueller-gis - maybe a good candidate for looking into the sized box discussion?

Reply:
When I removed the height of 50, the box resized to fit the the button. This seems to work.

Reply:
I don't actually know, but I wonder if SizedBox is the best Widget for the job here. Can it be done away with completely? Can the Center widget handle the necessary duties? Maybe there's some other flags we can pass to Center. Or maybe there's another widget that is simpler than SizedBox, something in the Container family.

Overall the goal is to figure out the simplest way to have a row of controls at the bottom, with a minimum height, that allows the Expanded around the MapView to take up all the slack.

Reply:
The `SizedBox` widget has been removed leaving only the `Center`. Still looks good.